### ISSUE TITLE
Illustrations

### ISSUE THREAD

**Issue by michael-77** [laurakwriting/john-muir_my-first-summer-in-the-sierra#4]
`<figcaption>` elements have inconsistent periods. Check to ensure that they all finish with a full stop.

The `<img>` `alt` element should describe the image in detail for those using screen readers, rather than serve the same function as `<caption>` - see https://standardebooks.org/contribute/semantics for section on images. They should be reviewed to make sure that they describe the image in that way.
For example:
```
<figure id="front-of-cathedral-peak">
	<img alt="This drawing shows the front of Cathedral Peak." src="../images/front-of-cathedral-peak.jpg" epub:type="z3998:illustration"/>
	<figcaption>Front of Cathedral Peak</figcaption>
</figure>
```
could become
```
<figure id="front-of-cathedral-peak">
	<img alt="A drawing of a tall rocky mountain with two peaks. At the foot of the mountain, there are fir trees and a lake." src="../images/front-of-cathedral-peak.jpg" epub:type="z3998:illustration"/>
	<figcaption>Front of Cathedral Peak</figcaption>
</figure>
```

I had thought when looking at the source code that quite a few of the photos and illustrations lend themselves to being full page elements and should be styled with the `full-page` class but they render very nicely on my kobo just the way they are so I would be happy to leave them alone from that point of view.  

Herbert W. Gleason (https://id.loc.gov/authorities/names/n50033099) and Charles S. Olcott (https://en.wikipedia.org/wiki/Charles_S._Olcott & https://id.loc.gov/authorities/names/nr94017193) should be added as contributors (illustrators) in content.opf

**Comment by laurakwriting** [comment]
Fixes to LOI, alt text, figure captions, and removal of endnotes (because the text is now in LOI) included in the following commit:

- dd50982 Fixes to figure captions, LOI, alt text, and removal of endnotes

**Comment by laurakwriting** [comment]
I've now added the following contributors (and metadata, not shown below) to content.opf:

```
<dc:contributor id="photographer-1">Herbert W. Gleason</dc:contributor>
<dc:contributor id="photographer-2">Charles S. Olcott</dc:contributor>
<dc:contributor id="cartographer">United States Geological Society</dc:contributor>
```
Fixes in this commit:
- 36f646f Adds photographers and cartographer contributors to content.opf

**Comment by laurakwriting** [comment]
Found a typo - needed to fix cartographer name, which should be:

United States Geological Survey 
(not "United States Geological Society")

Fix in the following commit:
- abd6df9 Fixing typo in content.opf for cartographer name

**Comment by laurakwriting** [comment]
Adding the *.source.jpg version of the map made me think of something - 

I probably should add similar *.source.jpg versions of all images. I have all of those larger (higher res) source files and can add them to the john-muir_my-first-summer-in-the-sierra/images folder if that sounds like a good idea.

**Comment by michael-77** [comment]
If you could do a review of https://github.com/laurakwriting/john-muir_my-first-summer-in-the-sierra/commit/dd5098247fed90efa249f675fbbf48b4416d2737 to change `alt` text which is non-descriptive or simply re-stating place names from the caption.  

> ## Images
> * All `<img>` tags are required to have an `alt` attribute that uses prose to describe the image in detail; this is what screen reading software will be read aloud.
>   * Describe the image itself in words, which is not the same as writing a caption or describing its place in the book.
>   * Alt text must be full sentences ended with periods or other appropriate punctuation. Sentence fragments, or complete sentences without ending punctuation, are not acceptable.
> 
> For example:
> ❌ `<img alt="The illustration for chapter 10" src="...">`
> ❌ `<img alt="Pierre's fruit-filled dinner" src="...">`
> ✔ `<img alt="An apple and a pear inside a bowl, resting on a table." src="...">`
> 
> Note that the `alt` text does not necessarily have to be the same as text in the image’s `<figcaption>` element. You can use `<figcaption>` to write a concise context-dependent caption. 

Specifically:
```
<figure id="the-north-and-south-domes">
	<img alt="A photograph showing three of Yosemities massive rock formations: Half Dome (also called South Dome), North Dome, and the Washington Column." class="full-page" src="../images/the-north-and-south-domes.jpg" epub:type="z3998:photograph"/>
	<figcaption>Half Dome (South Dome), North Dome, and Washington Column.</figcaption>
</figure>
```
is an example of an image which is hard to picture from the `alt` text.

I would agree that adding the *.source.jpg is a good plan.

I also note that `cover.source.jpg` appears twice in the repository (copies in both of the images folders by mistake).

I think we're almost there :-) Illustrations are painful.

**Comment by laurakwriting** [comment]
Thank you, Michael!

Sorry about the img alt texts, I think (and hope) I now understand the style they must have. For fixes, please see:
- 9c85cc2 Improvements to alt text for images

I also added the high-res source files for all of the images (and removed the extra cover.source.jpg). For those updates, please see:
- 05b2c85 Adds high-res JPG sources for all images, removes cover source placed…
- 27a14d4  Update manifest to remove cover.source.jpg

Let me know if any additional fixes are needed,
And thanks again for all your feedback, it's been very helpful,

Laura

**Comment by michael-77** [comment]
Those look really good. Thanks so much for going back over them. Much appreciated.

**Comment by michael-77** [comment]
I'm on mobile just now and will pull and Dona final check on build etc (always good to check on an independent machine) later today and then pass things on to Alex. 

Thanks again

Michael

**Comment by laurakwriting** [comment]
Sounds good, thank you,
Laura

**Comment by michael-77** [comment]
If you run `modernize-spelling` a final time then it will make a couple of changes to the new `alt` text.

`lint`, `semanticate`, `clean`, `build-images` and `typogrify` don't make any changes.

`build` runs cleanly and there are no errors from epubcheck. 

Once the `modernize-spelling` changes are in we should be good to go :-)

**Comment by laurakwriting** [comment]
Thank you!
Revisions in this commit:
- 8327d5a Ran modernize spelling one more time
